SimTracker/Common/interface/SiPixelChargeReweightingAlgorithm.h

```diff
@@ -669,7 +672,14 @@ inline bool SiPixelChargeReweightingAlgorithm::lateSignalReweight(const PixelGeo
     unsigned int chan = loopDigi->channel();
     if (loopTempSH.isInTheList(chan)) {
       float corresponding_charge = loopDigi->adc();
-      theDigiSignal[chan] += digitizerUtility::Amplitude(corresponding_charge, corresponding_charge);
+      if constexpr (std::is_same_v<AmplitudeType, digitizerUtility::Ph2Amplitude>) {
+        edm::LogError("SiPixelChargeReweightingAlgorithm")
+            << "Phase-2 late charge reweighting not supported (not sure we need it at all)";
+        return false;
+        //theDigiSignal[chan] += AmplitudeType(corresponding_charge, &hit, corresponding_charge, 0., hitIndex, tofBin);
```

Comment:
commented out code (after a `return` statement)

Reply:
will do in the next push